### PR TITLE
fix bullet lists and markdown link

### PR DESCRIPTION
**Pull request by aaronpk**

### CODE REVIEW

File: _source/_posts/2018-05-16-multifactor-authentication-4-challenges-faced-by-developers.md [hunk 1]
```diff
@@ -13,11 +13,12 @@ MFA is rapidly gaining in adoption, and more and more organizations are interest
...
-Store data about your users
-Perform password-based and [social login](https://developer.okta.com/authentication-guide/social-login/)
-Secure your application with (multi-factor authentication](https://developer.okta.com/use_cases/mfa/)
-And much more! Check out our [Product Documentation](https://developer.okta.com/documentation/)
+
+* [Authenticate](https://developer.okta.com/product/authentication/) and [authorize](https://developer.okta.com/product/authorization/) your users
+* Store data about your users
+* Perform password-based and [social login](https://developer.okta.com/authentication-guide/social-login/)
+* Secure your application with [multi-factor authentication](https://developer.okta.com/use_cases/mfa/)
+* And much more! Check out our [Product Documentation](https://developer.okta.com/documentation/)
```
Comment (leebrandt) [inline _source/_posts/2018-05-16-multifactor-authentication-4-challenges-faced-by-developers.md]: is this not lower-case too?

File: _source/_posts/2018-05-16-multifactor-authentication-4-challenges-faced-by-developers.md [hunk 2]
```diff
@@ -70,20 +74,24 @@ This is the most sophisticated, secure and hardest to implement approach. This a
...
 The flow can work in a variety of ways, but here are two examples:
-*Passwordless*
-You launch an application on your smartphone
-You see a screen waiting for your fingerprint
-You touch your finger to the fingerprint reader
-You are authenticated and enter the app
+
+**Passwordless**
+
+1.  You launch an application on your smartphone
```
Comment (mraible) [inline _source/_posts/2018-05-16-multifactor-authentication-4-challenges-faced-by-developers.md]: There's an extra space after 1., but maybe it's not rendered in the HTML?

File: _source/_posts/2018-05-16-multifactor-authentication-4-challenges-faced-by-developers.md (hunk 1 above)
Comment (mraible) [inline _source/_posts/2018-05-16-multifactor-authentication-4-challenges-faced-by-developers.md]: Didn't we agree this should be lower case?

File: _source/_posts/2018-05-16-multifactor-authentication-4-challenges-faced-by-developers.md (hunk 2 above)
Comment (leebrandt) [inline _source/_posts/2018-05-16-multifactor-authentication-4-challenges-faced-by-developers.md]: there's an extra space here. I don't think it'll matter once converted to HTML, but...

File: _source/_posts/2018-05-16-multifactor-authentication-4-challenges-faced-by-developers.md [hunk 3]
```diff
@@ -124,6 +132,6 @@ In this post, we’ve covered some of the approaches to multi-factor authenticat
 Happily, multi-factor authentication is at the very core of Okta’s business: providing the identity layer for companies, applications, and people. With just a few lines of code, you can add support for SMS, Google Authenticator, Okta Verify and many others to your application today. If you use the [Okta Sign-In Widget](https://github.com/okta/okta-signin-widget), no additional coding for MFA support is needed!
 
 To learn more about MFA with Okta, check out these resources:
-* [Our MFA Product Documentation](https://developer.okta.com/use_cases/mfa/)
+* [Our MFA product documentation](https://developer.okta.com/use_cases/mfa/)
```
Comment (mraible) [inline _source/_posts/2018-05-16-multifactor-authentication-4-challenges-faced-by-developers.md]: With all the other caps in this listing, caps seems kinda appropriate here.

**Pull request merged**